src/profile-logic/marker-schema.js

```diff
@@ -339,8 +340,27 @@ export function getLabelGetter(
 ): (MarkerIndex) => string {
   // Build up a list of label functions, that are tied to the schema name.
   const labelFns: Map<string, (Marker) => string> = new Map();
+  const markerNamePrefixRe = /^{marker.name}\s[-—]\s/;
   for (const schema of markerSchemaList) {
-    const labelString = schema[labelKey];
+    let labelString;
+    if (labelKey === 'copyLabel') {
+      // When copying a marker description, use the marker table label.
+      labelString = schema.tableLabel;
+      if (labelString && !markerNamePrefixRe.test(labelString)) {
+        // Ensure the label starts with the marker name.
+        labelString = '{marker.name} — ' + labelString;
+      }
+    } else {
+      labelString = schema[labelKey];
+      // The marker table used to not show the marker name, so all marker
+      // schemas included the marker name as the first part of their table
+      // label. Now that the marker table has a name column, we can remove
+      // this prefix to avoid duplication.
+      if (labelString && labelKey === 'tableLabel') {
+        labelString = labelString.replace(markerNamePrefixRe, '');
+      }
```

Comment:
Hmm this looks a bit hacky. I wonder if it's better to bump the profile version and write an upgrader for removing the old `marker.name`s from the start of the table tooltips.

@julienw do you have an opinion here?

Reply:
We discussed this a bit, I wasn't sure either.
If we do an upgrader this means we need to upgrade the gecko version and also change all currently existing occurrences in gecko. This sounds like more work.
So in the end I think this solution is easier and simpler. We can still remove existing occurrences in gecko, possibly using good first issues.

Reply:
Yeah, I agree that it's more work.
It would be nice to still upgrade the version in the end when we remove all the occurrences in the backend. But let's land this one without an upgrade then.